code/datums/looping_sounds/machinery_sounds.dm

```diff
@@ -157,3 +157,10 @@
 	volume = 20
 	falloff_distance = 2
 	falloff_exponent = 5
+
+///////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
```

Comment:
Don't do this, it's not helpful and hurts doc comments

Reply:
Okay, I'll remove all other such lines in this file then too.